codegenerator/cli/templates/static/codegen/src/Env.res

```diff
@@ -63,14 +63,14 @@ module Benchmark = {
     let shouldSavePrometheus = self =>
       switch self {
       | Prometheus => true
-      | JsonFile | Bool(_) => false
+      // Always save benchmarks in Prometheus in TUI mode
+      // This is needed for Dev Console profiler
+      // FIXME: This doesn't take into account the arg from the CLI
+      | _ => !tuiOffEnvVar
```

Comment:
So if the TUI is off and the user didn't ask to use Prometheus then it won't put the benchmarks in Prometheus?

I'm not so familiar with what the desired behavior is here, `shouldSavePrometheus` only affects additional metrics for the benchmarks, the standard prometheus metrics will still show?

Reply:
Yes, this is only for benchmarks. Previously, we only had it only for hosted service, but now it should be always enabled when you run with TUI on.

Reply:
Currently TUI mode is perceived as a dev mode. In the future we should have better differentiation.

Reply:
Yeah, I'm keen for a 'dev' mode that does lots more, like exposes an API that users can use to understand their indexer better (someting for the AIs to consume :D )